app/views/applications/show.html.erb

```diff
@@ -32,12 +33,20 @@
 </ul>
 
 <% if @application.archived %>
-  <p>This application has been marked as archived.</p>
+  <%= render "govuk_publishing_components/components/notice", {
+    title: "This application has been marked as archived.",
+    margin_bottom: 4,
+  } %>
 <% end %>
 
 <%= render 'status_notes', application: @application %>
 
-<h2 class="add-bottom-margin">Commit log</h2>
```

Comment:
Is there any CSS that needs deleting here?

Reply:
No, again it looks like a `govuk_admin_template` thing https://github.com/alphagov/govuk_admin_template/search?q=add-bottom-margin&unscoped_q=add-bottom-margin